docs/cheatsheet.rst

```diff
@@ -126,28 +126,34 @@ The steps are as follows:
 
 1. Create an issue on github.
 ::
-  hub issue create
-  <enter text>
 
-2. Start a new feature with hubflow named: :code:`issue-<issue # you created in step 1>-<description of work>`
+    hub issue create
+    <enter text>
+
+2. Start a new feature with hubflow named :code:`issue-<issue # you created in step 1>-<description of work>`
 ::
-  hub hf feature start issue-1-going-to-write-some-code
+
+    hub hf feature start issue-1-going-to-write-some-code
 
 3. Write code
 
 This is where the actual magic happens.
 
 4. Commit it
 ::
-  hub commit -a -m "hey look, real work!"
+
+    hub commit -a -m "hey look, real work!"
 
 5. Push it back up to github
 ::
-  hub push
+
+    hub push
 
 6. Convert the issue into a Pull Request
 ::
-  hub pull-request -b develop -i 1
+
+    hub pull-request -b develop -i 1
```

Comment:
The `-i <issue_num>` has been deprecated and doesn't work anymore sadly. I think we should remove this. The command doesn't fail, it just doesn't convert an issue to a PR anymore. It creates a new issue number.